pkg/reconciler/taskrun/taskrun.go

```diff
@@ -170,6 +170,9 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, tr *v1.TaskRun) pkgrecon
 	}
 
 	// Check for Pod Failures
+	// TODO:
```

Comment:
Could you please add a tracking issue for this.

Reply:
This should be deleted actually. Thanks for the catch!